app/src/main/java/com/sapuseven/untis/widgets/BaseWidget.kt

```diff
@@ -84,7 +85,11 @@ open class BaseWidget : AppWidgetProvider() {
 
 		val updateIntent = Intent(context, this.javaClass)
 				.setAction(AppWidgetManager.ACTION_APPWIDGET_UPDATE)
-		val pendingIntent = PendingIntent.getBroadcast(context, 0, updateIntent, PendingIntent.FLAG_UPDATE_CURRENT)
+		val pendingIntent = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+			PendingIntent.getBroadcast(context, 0, updateIntent, PendingIntent.FLAG_UPDATE_CURRENT + PendingIntent.FLAG_IMMUTABLE)
```

Comment:
Flags are combined using `|`
```suggestion
			PendingIntent.getBroadcast(context, 0, updateIntent, PendingIntent.FLAG_UPDATE_CURRENT | PendingIntent.FLAG_IMMUTABLE)
```